cloudrone/State state
int32 newstate
---
cloudrone/State state
uint32 error